### PR TITLE
feat: add how-to-integrate-autoware

### PR DESCRIPTION
**Pull request by kminoda**
Signed-off-by: kminoda <koji.m.minoda@gmail.com>

## Description

Add "Integrate Autoware with your vehicle" in how to guide section.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# How to integrate Autoware with your vehicle
+
+This page demonstrates how to intagrate Autoware with the real vehicle.
```
Comment (kenji-miyake) [inline docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md:3]: typo and a fix by Grammarly
```suggestion
This page demonstrates how to integrate Autoware with a real vehicle.
```

File: docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+# How to integrate Autoware with your vehicle
+
+This page demonstrates how to integrate Autoware with a real vehicle.
+
+## 1. Prepare your real vehicle hardware
+
+Prerequisites for the vehicle:
+
+- car-like or diff-drive vehicle
```
Comment (kenji-miyake) [inline docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md:9]: Probably you should capitalize each item?

https://developers.google.com/style/lists?hl=en
```suggestion
- Car-like or diff-drive vehicle
```

File: docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- onboard computer that satisfies the prerequisites (see [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#prerequisites))
+- the following devices attached
+  - actuator
+  - LiDAR
+  - inertial measurement unit (optional)
```
Comment (kenji-miyake) [inline docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md:14]: https://developers.google.com/style/procedures?hl=en#optional-steps
```suggestion
  - Optional: Inertial measurement unit
```

**Comment by kminoda** [comment]
Sorry, I mistakenly merged unrelated commits in this PR.
I will close this and reopen a new one.

**Pull request closed without merging**